tools/manage_clvm.py

```diff
@@ -291,35 +337,49 @@ def check(use_cache: bool) -> int:
         for exclude in unused_excludes:
             print(f"    {exclude}")
 
+    extra_hashes = [key for key in HASHES if key not in all_hash_stems]
+    if extra_hashes != []:
+        overall_fail = True
+        print()
+        print("Hashes without corresponding files:")
+        for extra_hash in extra_hashes:
+            print(f"    {extra_hash}")
+
     if use_cache and cache_modified:
         cache_path.parent.mkdir(parents=True, exist_ok=True)
         with cache_path.open(mode="w") as file:
             dump_cache(cache=cache, file=file)
 
+    if fix_hashfile_trailing_whitespace:
+        hashes_path.write_text(json.dumps(HASHES, indent=4, sort_keys=True) + "\n")
+
     return 1 if overall_fail else 0
 
 
 @main.command()
 def build() -> int:
     overall_fail = False
 
-    found_stems = find_stems(top_levels, suffixes={"clvm": clvm_suffix})
+    found_stems = find_stems(top_levels, suffixes={"clsp": clsp_suffix})
+    hash_stems = []
+    new_hashes = HASHES.copy()
 
-    print(f"Building all existing {clvm_suffix} files to {hex_suffix}:")
-    for stem_path in sorted(found_stems["clvm"]):
-        clvm_path = stem_path.with_name(stem_path.name + clvm_suffix)
+    print(f"Building all existing {clsp_suffix} files to {hex_suffix}:")
+    for stem_path in sorted(found_stems["clsp"]):
+        clvm_path = stem_path.with_name(stem_path.name + clsp_suffix)
```

Comment:
I deliberately left these because I think clvm is the best descriptor of everything as a whole.  These variables refer kind of collective to the chialisp/hex/hash.

Reply:
At least in this case it is specifically a path for a clsp file with a `.clsp` suffix.  Isn't that not a clvm file?